sdk/quota/arm-quota/test/quota_operations_test.spec.ts

```diff
@@ -45,6 +46,10 @@ describe("quota test", () => {
       subscriptionId,
       recorder.configureClientOptions({}),
     );
+    client1 = new AzureQuotaExtensionAPI(
+      credential,
+      recorder.configureClientOptions({}),
+    );
```

Comment:
Do we have a test case where the subscriptionId is used finally? If no, could you add one?

Reply:
Do you mean this one?
https://github.com/Azure/azure-sdk-for-js/pull/35947/files/a2ba3211527cdcc3fc0ede5c2551ef699c260d61..39e0d0aeeae82a21f144f4d993c9e91d5d2f68a2#diff-f0d79c74d57059149afaec2e02695c98d9444ecd7579345039bb99bd19969bf5R44-R48